keystore/src/entities/platform/generic/mls/encryption_keypair.rs

```diff
@@ -196,7 +188,7 @@ impl EntityBase for MlsEncryptionKeyPair {
         let len = ids.len();
         let mut updated = 0;
         for id in ids {
-            updated += transaction.execute("DELETE FROM mls_encryption_keypairs WHERE pk = ?", [id.as_slice()])?;
+            updated += transaction.execute("DELETE FROM mls_encryption_keypairs WHERE pk_sha256 = ?", [id.sha256()])?;
```

Comment:
Nitpick: I'd rename `updated` to `deleted`, since we're deleting rows.

Reply:
I'd like to deal with this when eliminating code duplication, since this would mean that we touch every file where the `Entity` trait is implemented.